resources/lib/library.py

```diff
@@ -238,6 +263,27 @@ def retrieveContent( self, content ):
             items = self.checkForFolder( items )
         else:
             items = []
+
+        # Add widget information for video/audio nodes
+        if content == "video" or content == "music":
+            # Video nodes - add widget information
+            for listitem in items:
+                path = listitem.getProperty( "path" )
+                if path.lower().startswith( "activatewindow" ):
```

Comment:
...and
```if not path.lower().startswith( "activatewindow" ): continue ```for these cases to save some levels of indentation. Probably matter of taste though.